### DISCUSSION TITLE
【提醒】目前发现浏览绳网遇到的常见问题以及（暂时的）解决方法

### DISCUSSION THREAD

**Discussion by wugan-bot156** [share121/inter-knot#1105] [其他]
![无标题5_20230316214410(1)](https://github.com/user-attachments/assets/7d7eb403-eec5-41be-bc91-e141e0f19c0b)

问题1.目前绳网界面没有搜索和查看历史记录的功能，需要到GitHub的discussions翻找
          解决方法：绳匠们在discussions处给感兴趣的贴子标星方便查找，然后用#＋数字开传送门。举例：#540，这是我关于解决关闭邮箱提醒的贴子。 在绳网界面点击回复后会进入GitHub页面，把页面拉到最上面，数字就在标题旁边。

问题2.有些人包括我遇到过无法直接通过右键保存贴子的图片的情况
         解决方法：右键后选择编辑图像，然后就可以在左上角保存图片。

**Comment by Y-0930** [comment]
我全靠邮箱定位帖子😓
